website/components/News.tsx

```diff
@@ -21,10 +21,10 @@ const newsContainer = css`
   padding-bottom: ${spacing[4]}px;
 
   ${mq({
-    marginTop: [`${spacing[4]}px`, 'unset'],
-    marginBottom: [`${spacing[4]}px`, 'unset'],
-    paddingLeft: [`${spacing[4]}px`, 'unset'],
-  })}
+  marginTop: [`${spacing[4]}px`, 'unset'],
+  marginBottom: [`${spacing[4]}px`, 'unset'],
+  paddingLeft: [`${spacing[4]}px`, 'unset'],
+})}
```

Comment:
This formatting change doesn't look correct.

Reply:
This should never have made it in here, was an accident. I'm removing the whole file.

Reply:
👍